kuma/users/jinja2/users/ban_user.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base.html" %}
+
+{% block body_attributes %}{% endblock %}
+
+{% block bodyclass %}user ban-user{% endblock %}
+
+{% block title %}Ban {{ user_to_ban }}{% endblock %}
+
+{% block site_css %}
+  {{ super() }}
+  {% stylesheet 'users' %}
+{% endblock %}
+
+{% block content %}
+<section id="content">
+<div class="wrap">
+  <section id="content-main" class="full">
+    <section id="user-head" class="column-container column-container-reverse vcard user-head">
+      <div class="column-10">
+        <h1 class="user-title">
+          {{ _('Why are you banning') }} {{user_to_ban }}?
```

Comment:
Please use [newstyle gettext](http://jinja.pocoo.org/docs/dev/extensions/#newstyle-gettext) on the complete sentence, to handle languages with different word order, leading '¿' in Spanish, etc.

```
{{ _('Why are you banning %(user)s?', user=user_to_ban) }}
```